sdk/communication/azure-communication-sms/azure/communication/sms/_generated/aio/operations/_sms_operations.py

```diff
@@ -12,7 +12,7 @@
 from azure.core.pipeline import PipelineResponse
 from azure.core.pipeline.transport import AsyncHttpResponse, HttpRequest
 
-from ... import models
+from ... import models as _models
```

Comment:
Not following why this changed?

Reply:
I think its part of the newer autorest version. The [identity](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication/azure-communication-identity/azure/communication/identity/_generated/operations/_communication_identity_operations.py#L15) and [phone number](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_generated/operations/_phone_number_administration_operations.py#L16) packages in python also do it this way.